cmake/Re2cGenDocs.cmake

```diff
@@ -29,12 +30,11 @@ function(re2c_gen_help source target bootstrap)
         add_custom_command(
             OUTPUT "${target}"
             COMMAND "${CMAKE_COMMAND}" -E make_directory ${targetdir}
-            COMMAND "${RST2MAN}" "${source}" "${target}.1"
-            COMMAND "${re2c_genhelp}" "${target}.1" "${target}"
+            COMMAND "${PYTHON}" "${re2c_rst2txt}" --variable-name=help "${source}" "${target}"
```

Comment:
Is `--variable-name=help` used in rst2txt.py? I don't see the arguments either used or forwarded.

Reply:
Command-line parsing and handling is done by the Docutils `publish_cmdline` function. The `-variable-name` option gets forwarded to the txt writer. See the `settings_spec` variable.